thirdparty/libwebsockets/lws_config.h

```diff
@@ -78,6 +78,7 @@
 
 /* Build with support for ipv6 */
 /* #undef LWS_WITH_IPV6 */
+#define LWS_WITH_IPV6
```

Comment:
Please add `// -- GODOT start --` and `// -- GODOT end --` comments around it, so that we know that we changed this on purpose. Otherwise it will likely be reset next time with sync with upstream's default config.